apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2024-03-29T14:30:36Z"
-  build_hash: e8df4d5a4b86dea0e227786c2c3d213e5aeda97a
-  go_version: go1.22.0
-  version: v0.33.0
```

Comment:
Can we regenerate using the controller using code-gen 0.33.0 (released a few days ago)

Reply:
Yes sure thing